woocommerce/woocommerce-ios
# Simple Payments: User payment link provided by the remote API

Closes: #6087

# Why 

Previously, we were building orders payment links inside the app, but this proved to not be reliable as the merchants can change different parts of that URL in their store settings.

As of `WC 6.4` stores will start delivering the `payment_url` field inside the order entity.

This PR makes sure that the payment link shared is comming from the remote API and not built locally.

# How 

- Removes `PaymentLinkBuilder` type.

- Updates `SimplePaymentsMethod` view to remove the beta tag annotation.

- Update `SimplePaymentsMethodsViewModel` and `SimplePaymentsSummaryViewModel` to pass along the order `payment_link` from the moment it is created.

# Demo

### WC < 6.4

https://user-images.githubusercontent.com/562080/162231977-b98555c8-f26a-48d6-82d0-ceac4fc3a1dd.mov

### WC >= 6.4

https://user-images.githubusercontent.com/562080/162232721-858bd9bb-eba1-4c00-854e-5bb51d119604.mov


# Testing Steps

- Make sure you have updated your store to the `6.4.0-rc.1` version.
- Start a "Simple Payment" and go all the way until the "Payment Method" screen.
- See that the payment link is available and don't show any beta indicator.

---
